bower.json

```diff
@@ -12,7 +12,7 @@
   "description": "jQuery Split Pane plugin",
   "license": "MIT",
   "dependencies": {
-    "jquery": "1.11.1"
+    "jquery": ">=1.11.1"
```

Comment:
yes 2.1.4 works fine!, also i think we can test it later on 3.0.0 alpha :)
Also pls look at this: https://github.com/shagstrom/split-pane/issues/14
i writed a some messages to your email from http://www.dreamchain.se